app/src/main/java/com/droidhats/campuscompass/adapters/SearchAdapter.kt

```diff
@@ -0,0 +1,100 @@
+package com.droidhats.campuscompass.adapters
+
+import android.graphics.Color
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.EditText
+import android.widget.ImageButton
+import androidx.appcompat.widget.SearchView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.droidhats.campuscompass.R
+import com.droidhats.campuscompass.models.GooglePlace
+import com.droidhats.campuscompass.models.IndoorLocation
+import com.droidhats.campuscompass.models.Location
+import com.droidhats.campuscompass.views.SearchFragment
+import com.droidhats.campuscompass.views.SearchFragment.Companion.isNavigationViewOpen
+import kotlinx.android.synthetic.main.search_suggestion_recycler_item.view.*
```

Comment:
is it possible to not * import for this package?

Reply:
Definitely!